lib/wanda_web/controllers/v1/operation_controller.ex

```diff
@@ -37,10 +37,13 @@ defmodule WandaWeb.V1.OperationController do
     ]
 
   def index(conn, params) do
-    operations = Operations.list_operations(params)
-    enriched_operations = Enum.map(operations, &Operations.enrich_operation!(&1))
+    operations =
+      params
+      |> Operations.list_operations()
+      |> Enum.map(&Operations.enrich_operation!(&1))
+      |> Enum.map(&Operations.compute_aborted(&1, DateTime.utc_now()))
```

Comment:
Have you considered computing whether an operation is aborted or not (based on the time) in the context (`list_operations` and `get_operation` functions).

It would add extra consistency, imho, as we'd always have that logic applied when listing or getting a single operation.

Reply:
Yes, initially I did.
I would need to change the signature to add the current utc time, which looks a bit out of place (it could be an optional argument as well, like dependency injection only for tests).
The `list_operations` already has a default argument as well, and I could get the time from there, but it looks a bit hacky.
Otherwise testing  becomes impossible
Anyway, I don't have strong opinions

Reply:
I would prefer having them separated and composed when needed like in the controller which is the de-facto integration point instead of polluting the signature

Reply:
Up to you guys.

Anyway the signature might not need to be necessarily changed. We could rely on a clock/datetime behaviour which is mockable in tests. Something similar to what we do in web with `Trento.Support.DateService`

Reply:
But you still need to pass the configurable `DateService` to the function. How do you run it otherwise?